libpod/oci_conmon_linux.go

```diff
@@ -1279,19 +1278,6 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		// conmon not having a pid file is a valid state, so don't set it if we don't have it
 		logrus.Infof("Got Conmon PID as %d", conmonPID)
 		ctr.state.ConmonPID = conmonPID
-
-		// Send the MAINPID via sdnotify if needed.
-		switch ctr.config.SdNotifyMode {
-		case define.SdNotifyModeContainer, define.SdNotifyModeIgnore:
```

Comment:
looking at `(c *Container) start()` the condition there is `SdNotifyMode != define.SdNotifyModeIgnore`.
What is correct? I think SdNotifyModeContainer means that conmon is sending the MAINPID so podman should not send.

Reply:
I probably missed something, thanks.  I'll double check.

Reply:
It looks correct to me.

With `--sdnotify=container` conmon is still the MAINPID but the container must send the ready message.  That is also the logic in `(*Container).start()`

Reply:
ok just wanted to check what the correct behaviour is since the differ